scripts/symbolize_mem_dump.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python3
+#
+# This file is open source software, licensed to you under the terms
+# of the Apache License, Version 2.0 (the "License").  See the NOTICE file
+# distributed with this work for additional information regarding copyright
+# ownership.  You may not use this file except in compliance with the License.
+#
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+# Copyright (C) 2023 ScyllaDB
+
+import addr2line
+import fileinput
+import sys
+
+# Expects N lines of format:
+# size count backtrace
+# For example:
+# 12344 2 0x1234 0x1456 0x34667
+
+# To be invoked like:
+# something_producing_output_as_above | symbolize_mem_dump.py /path/to/binary | flamegraph.pl - > flamegraph.svg
```

Comment:
I think it's worth looking for something better than `flamgraph.pl`. Although it's the OG flameframe, I'm pretty sure it's been passed by others in the meantime with better ergonomics. BTW do the dynamic SVG features in the output like search and zoom actually work for you? They don't for me in the examples you posted in the issue (though maybe that's a github thing).